Multiple to Single Action Sound Effect

talios0 opened the pull request:
Instead of allowing an array of sound effects for each action, the character is now limited to a single sfx.

The state sfx are still arrays, allowing for multiple sound effects to be included.

The test character was also modified to accommodate this change.

Apexal reviewed (approved):
LGTM!

Apexal commented on src/interfaces/action.ts:
```diff
@@ -28,5 +28,5 @@ export interface Action {
   animation: AnimationFrame[];
   /** Possible sound effects for the action. One is chosen randomly each time the move is performed. */
-  soundEffects: SoundEffect[];
+  soundEffects: SoundEffect;
```
This property name should be changed to `soundEffect` and the type should be nullable (`| null`).

talios0 replied:
Good point!

talios0 commented:
Just changed the "soundEffects" property from within action to be called "soundEffect" and made it nullable. I have also gone and updated the character.json inside constants to match. One action has info and the other is null.
